Dockerfile

```diff
@@ -4,55 +4,83 @@ FROM alpine:3.4
 # artisanally hand-rolling curl and the rest of our stack we'll just use
 # Alpine so we can use `docker build`.
 
-RUN apk add --update curl
+RUN apk add --update \
+    bash \
```

Comment:
It turns out the `-n` test in the `prestart.sh` is bash-specific. Of all the choices I had to fix it, I decided to add bash.